pkg/crawler/crawler.go

```diff
@@ -586,20 +581,28 @@ func (c *Crawler) generateLicenseFile(client http.Client, licenseFileName string
 
 	}
 
-	// download license url contents
-	resp, err := client.Get(licenseMeta.URL)
+	// get license url contents
+	resp, err := c.http.Get(licenseMeta.URL)
 	if resp == nil {
+		log.Printf("Err! License Meta URL not accessible %s", licenseMeta.URL)
```

Comment:
Do we need this. If the user has added a url that doesnt exists this will flood the logs. I see it here https://github.com/deepfactor-io/javadb/actions/runs/10205131805/job/28235210371